layout/layout_test.go

```diff
@@ -157,7 +157,7 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 					it("sets the initial state from a linux/arm base image", func() {
 						existingLayerSha := "sha256:5a0b973aa300cd2650869fd76d8546b361fcd6dfc77bd37b9d4f082cca9874e4"
 
-						img, err := layout.NewImage(imagePath, layout.FromBaseImage(testImage))
+						img, err := layout.NewImage(imagePath, layout.FromBaseImage(testImage), layout.WithMediaTypes(imgutil.OCITypes))
```

Comment:
I am a little confuse now. 

The test is using the following image: `arm64v8/busybox@sha256:50edf1d080946c6a76989d1c3b0e753b62f7d9b5f5e66e88bef23ebbd1e9709c` which is using `Docker Media-Types`

Adding `layout.WithMediaTypes(imgutil.OCITypes)` I will expect to *override* it and my image should have `OCI Media-Types`.  Should line 202 doesn't change, right?

Reply:
I think you need to expand the diff. Line 162 did _not_ change because of what you said. Line 202 is the result of line 200, which did _not_ use the `WithMediaTypes` override. I sort of mixed and matched these existing tests with OCI/Docker. Let me know if that doesn't make sense!

Reply:
Ohhh . sorry, let me check

Reply:
Sorry about that, you are right!! I was checking a bad line.